internal/models/tokens.go

```diff
@@ -31,3 +31,9 @@ type CustomClaims struct {
 	// https://datatracker.ietf.org/doc/html/rfc7519#section-4.1
 	jwt.RegisteredClaims
 }
+
+// TokenInfo defines a TokenInfo struct.
+type TokenInfo struct {
+	UID  string `json:"userid,omitempty"`
+	Name string `json:"name,omitempty"`
```

Comment:
Those fields can be empty? If they can't, please remove `omitempty`.

Reply:
See [https://pkg.go.dev/encoding/json#Marshal](https://pkg.go.dev/encoding/json#Marshal)